components/kserve/kserve_config_handler.go

```diff
@@ -25,18 +25,18 @@ const (
 func (k *Kserve) setupKserveConfig(ctx context.Context, cli client.Client, logger logr.Logger, dscispec *dsciv1.DSCInitializationSpec) error {
 	// as long as Kserve.Serving is not 'Removed', we will setup the dependencies
 
+	defaultDeploymentMode := k.DefaultDeploymentMode
+	if defaultDeploymentMode == "" {
+		defaultDeploymentMode = Serverless
+	}
+	disableIngressCreation := true
+	if k.RawRouteCreation == operatorv1.Managed {
```

Comment:
what would happen(assume the new field is called "rawRouteCreation" for now) , when user set
```
kserve:
      managementState: Managed
      defaultDeploymentMode: RawDeployment
      rawRouteCreation: Removed
      serving:
        ingressGateway:
          certificate:
            type: OpenshiftDefaultIngress
        managementState: Managed
        name: knative-serving
```